src/main/webapp/components/search-forms/search-forms.stories.tsx

```diff
@@ -44,10 +44,10 @@ const startingForms = [
   },
 ]
 
-const userAttributes = {
-  email: 'admin@test.com',
-  roles: ['admin'],
-}
+// const userAttributes = {
+//   email: 'admin@test.com',
+//   roles: ['admin'],
+// }
```

Comment:
delete?

Reply:
I'm hoping to uncomment this later, once I have the usePermissions hook able to take in a user through context